packages/metal-quartz-link/src/MetalQuartzLink.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import Component from 'metal-component';
+import MetalQuartzIcon from 'metal-quartz-icon';
+import Soy from 'metal-soy';
+import { core, object } from 'metal';
+import { validators } from 'metal-state';
+
+import templates from './MetalQuartzLink.soy.js';
+
+/**
+ * Implementation of the Metal Quartz Link.
+ */
+class MetalQuartzLink extends Component {
+	/**
+	 * @inheritDoc
+	 */
+	created() {
+		MetalQuartzLink.instances.push(this);
```

Comment:
This probably shouldn't go in here. I don't know if Metal has something that keeps track of components that have been rendered on the page. It's really difficult to access the state of a component. It's something I added to keep track of the components being rendered on the page so developers can have some kind of access.